packages/atlas/src/components/_channel/ChannelLink/ChannelLink.styles.ts

```diff
@@ -29,7 +29,6 @@ type HandleProps = {
 }
 
 const secondaryTextCss = `
-  color: ${oldColors.gray[200]};
```

Comment:
I guess we can remove `secondaryTextCss`, since it's not needed anymore

Reply:
It came back :)

Reply:
All right, it's removed!